meta-rpi-extras/classes/dualpart_image-rpi.bbclass

```diff
@@ -0,0 +1,116 @@
+inherit sdcard_image-rpi
```

Comment:
A lot of copying here. I have not looked at details yet... if it is not possible to extend `meta-raspberrypi/classes/sdcard_image-rpi.bbclass` we must at least add a header with copyright and license information.

I assume it is hard/impossible to extend? If so, perhaps a blurb at the top about what changes have been made as well and where the original function comes from.

Reply:
yes its not possible to just extend by appending. The required changes are scattered in the function, so unfortunately have to override and copy everything + updates/additions. Will add comments in the code to explain this.
Yes, I was also thinking about the license thing.